serverless/serverless-release-notes.adoc

```diff
@@ -7,7 +7,7 @@ include::_attributes/serverless-document-attributes.adoc[]
 
 toc::[]
 
-This topic includes information about new features, changes, and known issues that pertain to the latest {ServerlessProductName} release on {product-title}.
+Release notes contain information about new and deprecated features, breaking changes, and known issues. The following release notes apply for the most recent {ServerlessProductName} releases on {product-title}.
```

Comment:
Breaking changes? not sure what that means.

Reply:
It's a pretty common term that most developers would understand https://en.wiktionary.org/wiki/breaking_change

Reply:
ACK 👍 Thanks for explaining :)